charts/taraxa-node/values.yaml

```diff
@@ -0,0 +1,233 @@
+# Default values for taraxa-node.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  create: true
+
+serviceAccounts:
+  pythontester:
+    create: true
+    name:
+
+image:
+  repository: gcr.io/jovial-meridian-249123/taraxa-node
+  tag: latest
+  pullPolicy: IfNotPresent
+
+config:
+  # integer, 1=Mainnet, 2=Testnet, 3=Devnet)
+  # 100 for default helm test
+  network: "100"
+  extraArgs: []
+  
+  # Default keys, VRFs and address for kube testing
+  node:
+    keys:
+      - "c2bae6546061feb384742797cda745cddbb8552fdd044206053844dfacff8ae2"
+    vrfs:
+      - "590d5ff3d577ebca38c1f29e4798e8ce4ab9bd826ea7bccaba8c1d1bdaf5ff94ed07a2832b5a6077dd0147eae950d170ca8777dd3e8c9a01a035833fdc86ffcc"
+  consensusnode:
+    explorerDelegationPrivateKey: ""
+    keys:
+      - "badf7196e18f653130564fd2f27419bff36194094057a69053bbe3a83a97b4fe"
+    vrfs:
+      - "c7c32f136cf4529471905a6b775ad82a076a5b5d3160b76ba683c743b8a852cff06560102e3dbab2e8b62082611dfc378c90336d01c0a7fd2a1a7bb88fb63478"
+  bootnode:
+    keys:
+      - "45dc56636faf97230f557e16345055f5839dad25f4b3f6f88a02add24b4a00fc"
+    vrfs:
+      - "040950c16dcd7b6578ad32c93138d948dcccdc08a5d5e809eeecff59cd39c304e624bff2da26c0cfe512eb77df399b2ad676e1815e407b37d4c6ca4de35e37cd"
+    addresses:
+      - "31a472957604e93a77144b434162410f173f3962c950d7b8f9145a42254310db8244383d03f759159092232e8ef85f73806b0380a5e520e536158bb11037ec56"
+
+secrets:
+  enabled: true
+
+slack:
+  enabled: false
+  token: xoxb-XXXXXX-XXXXXX-XXXXXX
+  channel: channel
+
+nameOverride: ""
+fullnameOverride: ""
+
+domain: "default.example.com"
+# Add wild certs for this
+# wildCertDomainSecret: ""
+
+nodeSelector: {}
```

Comment:
Not having default values here, will break the actual helm test we currently run on PRs. (if we remove the files from the other repo)
I will add more details on the PR comment itself.

Reply:
In default helm chart we need to have default values like:
```
nodeSelector: {}
```
becasue, this is supposed to be available at some point for community users. 

Everyone should be able overwrite this with own values.

And we are overwriting this in gitops on "higher" level (cluster).

So far, this deployment chart will be used by `devnet` deployment. `testnet` and `mainnet` will still be using old chart repo.

- `devnet`: will use this chart repo
- `prnet`: will use old chart repo
- `testnet`: will use old chart repo
- `mainnet`: will use old chart repo

Such temporary solution will be untill devnet is stable enough, and everyone from ledger team know how to use. Than we will redeploy testnet and mainnet

Reply:
yeah, I know we should have default values
But I was just saying currently, it would break if we move circleci tests
if we don't plan on changing this now, this is fine.